app/js/components/subAddress/addressForm.jsx

```diff
@@ -194,20 +194,20 @@ export default class AddressForm extends React.Component {
           </div>
         </div>
         <div className="preffered">
-          {this.props.address.preferred ?
```

Comment:
Instead of chaining this much. Why don't you destructure by doing something like this `const { address: { preferred } } = this.props;` so you can use `preferred` instead of `this.props.address.preferred `

Reply:
Hy @joel-ace thanks for the feedback . The preferred tag is no longer gotten this way. It is got through this.state.preferred. I also wont advice destructuring here as it might get confusing. [check this](https://stackoverflow.com/questions/42307735/destructuring-state-in-react-render)

Reply:
Okay cool.